FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -15,7 +15,7 @@ public final class CoreDataFeedStore: FeedStore {
 		let modelName: String
 	}
 
-	public init(storeURL: URL) throws {
+	init(storeURL: URL) throws {
```

Comment:
This initialiser should be `public` to be visible from outside of the module. Otherwise it would not be available for usage by module's clients.

Please make it `public`.
```suggestion
	public init(storeURL: URL) throws {
```